src/sipreg/reg.c

```diff
@@ -446,8 +455,9 @@ int sipreg_send(struct sipreg *reg)
 int sipreg_register(struct sipreg **regp, struct sip *sip, const char *reg_uri,
 		    const char *to_uri, const char *from_name,
 		    const char *from_uri, uint32_t expires,
-		    const char *cuser, const char *routev[], uint32_t routec,
-		    int regid, sip_auth_h *authh, void *aarg, bool aref,
+		    const char *cuser, const char *cparams,
```

Comment:
This function is deprecated.
Please add a setter for cparams!

Reply:
Done